src/app/pages/candidate-details/candidate-details.page.html

```diff
@@ -1,8 +1,8 @@
 <ion-header>
     <ion-toolbar mode="ios">
-        <ion-title class="ion-text-center">Test details - {{ pageState.name }}</ion-title>
+        <ion-title id="title" class="ion-text-center">Test details - {{ pageState.name }}</ion-title>
         <ion-buttons slot="start">
-            <ion-button (click)="dismiss()">
+            <ion-button id="close" (click)="dismiss()">
```

Comment:
same with this one, "close-button" instead of just "close"

Reply:
Change to reflect commit